src/coreclr/jit/compiler.cpp

```diff
@@ -5322,6 +5332,61 @@ void Compiler::compCompile(void** methodCodePtr, uint32_t* methodCodeSize, JitFl
 #endif // FUNC_INFO_LOGGING
 }
 
+#if FEATURE_LOOP_ALIGN
+
+//------------------------------------------------------------------------
+// bbPlaceLoopAlignInstructions: Iterate over all the blocks and determine
+//      the best position to place the 'align' instruction. After 'jmp' are
+//      preferred over block before loop and in case there are multiple blocks
```

Comment:
```suggestion
//      the best position to place the 'align' instruction. Inserting 'align' instructions after an unconditional branch is
//      preferred over inserting in the block before the loop. In case there are multiple blocks
```